com.protonvpn.ProtonVPN.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+    <description>
+        <p>ProtonVPN is a secure and free VPN, now featuring better connection stability, faster
+            speeds, and even more security features.</p>
+        <p>ProtonVPN is the world's ONLY free VPN service that respects your privacy and is safe
+            to use. Millions of people use ProtonVPN because:</p>
+        <ul>
+            <li>We do not record your browsing history</li>
+            <li>We do not display privacy-invading ads</li>
+            <li>We do not sell your data to third parties</li>
+            <li>We do not limit how much you can download</li>
+        </ul>
+        <p>ProtonVPN is created by the CERN scientists behind ProtonMail, the world's largest
+            encrypted email service with 20 million users, including many activists and journalists
+            such as Reporters Without Borders.</p>
+        <p>ProtonVPN’s Free VPN Service includes:</p>
+        <ul>
+            <li>STRONG ENCRYPTION - Your data is protected with AES-256 and 4096 RSA.</li>
+            <li>SWISS-BASED - Your data is protected by some of the strongest privacy laws in the
+                world.</li>
+            <li>NO LOGS. EVER - We never log or share user data. Even if authorities request
+                information, we have nothing to share.</li>
+            <li>PERFECT FORWARD SECRECY - Your encrypted traffic cannot be captured and decrypted
+                later.</li>
+            <li>MULTI-PROTOCOL SUPPORT - ProtonVPN supports secure VPN protocols: WireGuard and
+                OpenVPN.</li>
+            <li>DNS LEAK PROTECTION - We encrypt your DNS to ensure your browsing activity cannot be
+                exposed through DNS queries.</li>
+            <li>ALWAYS-ON VPN/KILL SWITCH - Protect your IP address from being leaked through
+                accidental disconnects.</li>
+            <li>FULL DISK ENCRYPTION - Our VPN servers are fully encrypted to protect your data.</li>
+            <li>ANONYMOUS - You do not need to share any personal data to use ProtonVPN.</li>
+            <li>SPLIT TUNNELING SUPPORT - This advanced feature allows you to select what traffic
+                goes through the VPN tunnel.</li>
+            <li>BYPASS BLOCKING - Smart protocol selection automatically overcomes VPN bans and
+                unblocks censored content.</li>
+            <li>TRUSTED - Our code has undergone a security audit and is open source.</li>
+            <li>MULTIPLE PLATFORM SUPPORT - Easy to use on Android, iOS, Windows, macOS, Linux, and
+                more.</li>
+        </ul>
+        <p>Note: Free users can only connect one device to Free servers in Japan, the Netherlands,
+            and the US.</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <image>
+                https://images.prismic.io/gatsby-landing-pages/e6a71a70-741e-44ed-a947-439c0ea92801_secure-core.png</image>
```

Comment:
Is it possible to use a screenshot from an official source? If not possible screenshots should not include desktop backgrounds (but should include window shadows).